### PR TITLE
Handle 'No more variables left in this MIB View ' in snmp_facts

### PR DESCRIPTION
**Pull request by ernst-s**
##### SUMMARY

 Fixes issue ansible/ansible#49044
##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

snmp_facts module
##### ADDITIONAL INFORMATION

Instead of checking for the text in some of the OID's this skips all values which are an instance of EndOfMibView.
This resolves the issue. Previous workaround in the issue report resolved the crash but created some other problems by only removing the EndOfMibView from some OID's. 

```paste below

```

### CODE REVIEW

File: changelogs/fragments/613-snmp_facts-EndOfMibView [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - snmp_facts module - Skip EndOfMibView values in snmp_facts.py (https://github.com/ansible/ansible/issues/49044)
```
Comment (felixfontein) [inline changelogs/fragments/613-snmp_facts-EndOfMibView:2]:
```suggestion
  - snmp_facts - skip ``EndOfMibView`` values (https://github.com/ansible/ansible/issues/49044).
```

Comment (ernst-s) [reply changelogs/fragments/613-snmp_facts-EndOfMibView]: Commited the changed changelog fragment

File: changelogs/fragments/613-snmp_facts-EndOfMibView.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - snmp_facts module - skip ``EndOfMibView`` values (https://github.com/ansible/ansible/issues/49044).
```
Comment (Andersson007) [inline changelogs/fragments/613-snmp_facts-EndOfMibView.yml:2]:
```suggestion
  - snmp_facts - skip ``EndOfMibView`` values (https://github.com/ansible/ansible/issues/49044).
```
there is a small mismatch between changelog examples and the doc statement
```When writing a changelog entry that pertains to a particular module, start the entry with - [module name] - and the following sentence with a lowercase letter.```
i made a PR to fix https://github.com/ansible/ansible/pull/70551

**Comment by felixfontein** [comment]
@ernst-s thanks for fixing this!

**Pull request merged**